src/Utilities/helpers.js

```diff
@@ -28,14 +28,20 @@ export function parseCvssScore(cvssV2, cvssV3) {
         (cvssV3 && parseFloat(cvssV3).toFixed(1)) ||
         (cvssV2 && `${parseFloat(cvssV2).toFixed(1)} (v2)`) ||
         'N/A'
-    )
+    );
+}
+
+export function processDate(dateString) {
+    const date = new Date(dateString);
+    const dateFormat = require('dateformat');
+    return (date instanceof Date && !isNaN(date) && dateFormat(date, 'd mmm yyyy')) || 'N/A';
```

Comment:
Question - Does "d mmm yyyy" handle "19 Feb 2019" (aka dd mmm yyyy)? Does it go "07" or "7"?

Reply:
I believe `d mmm yyyy` is going to eliminate the leading zero.

Reply:
Double checked with our content person and it should be 07 Feb 2019 - hopefully a quick fix? :)

Reply:
Fixed